common/conf/env/DEVINFRA.properties

```diff
@@ -114,7 +114,6 @@ pa.rugby.api.key=none
 
 # DCR
 rendering.endpoint=http://localhost:3030
```

Comment:
While we're here are `rendering.endpoint` and `misc-rendering.baseURL` used for anything? I couldn't find references.

Reply:
I'm assuming `misc-rendering` was pre-emptively added, because we're considering adding a `misc-rendering` service[^1]?

[^1]: https://github.com/guardian/dotcom-rendering/issues/9325